resources/local/chainspec.toml.in

```diff
@@ -117,6 +117,8 @@ refund_handling = { type = 'no_refund' }
 #                 administrator accounts
 #   'burn': fees are burned
 fee_handling = { type = 'no_fee' }
+# The proportion of baseline rewards going to reward finality signatures specifically.
```

Comment:
@EdHastingsCasperAssociation I think the comment here was copy pasted by mistake from a rewards parameter